src/anyOf.ts

```diff
@@ -0,0 +1,3 @@
+export default function anyOf(input: string) {
```

Comment:
I think this should accept an array of (characters | 2 length arrays). The two-length arrays would be a tuple of start and end characters. Here's an example:

```js
anyOf(['a', 'b', 'c', ['0', '9'], ['A', 'Z']])
```
Implementation stub:

```ts
const charOrRange = string | [string, string];
export default function anyOf(input: Array<charOrRange>) {
  …
}
```

Maybe we could force the strings in `charOrRange` to be of length one with fancier typescript constructs.